front/temporal/labs/utils/google.ts

```diff
@@ -0,0 +1,127 @@
+import type { ModelId } from "@dust-tt/types";
+import { google } from "googleapis";
+
+import type { Authenticator } from "@app/lib/auth";
+import { getGoogleAuthFromUserTranscriptsConfiguration } from "@app/lib/labs/transcripts/utils/helpers";
+import type { LabsTranscriptsConfigurationResource } from "@app/lib/resources/labs_transcripts_resource";
+import type { Logger } from "@app/logger/logger";
+
+export async function retrieveRecentGoogleTranscripts(
+  {
+    auth,
+  }: {
+    auth: Authenticator;
+    userId: ModelId;
+  },
+  logger: Logger
+) {
+  const googleAuth = await getGoogleAuthFromUserTranscriptsConfiguration(auth);
```

Comment:
Function name is wrong, this should be `getGoogleAuth(auth)` or something similar.

Reply:
Well, first thing in that function is : 
```const transcriptsConfiguration =
    await LabsTranscriptsConfigurationResource.findByUserWorkspace({
      auth,
    });```

And then from there getting the google auth from nango. Doesn't it make sense to keep the name?

Reply:
Sometimes the implementation details of the function has to leak in its name if it's going to be doing something dangerous or costly, but in that case, it's not.

Reply:
But the function can't be used generically in the rest of the code though, if I just name it "getGoogleAuth" it might get used somewhere else that doesn't relate to labsconfiguration. 
Should I pass the labstranscriptsconfig instead of the auth to keep a non-confusing name?